build.settings

```diff
@@ -11,11 +11,109 @@ settings = {
 			UIPrerenderedIcon = true, -- set to false for "shine" overlay
 		}
 	},
-	
-	plugins = {
-        ["plugin.OneSignal"] =
+
+    android =
```

Comment:
These `build.settings` changes won't be needed for a public build of the OneSignal Corona SDK. Can we place these SDK dev only setting somewhere else?